src/makeRobotAccountant.js

```diff
@@ -6,7 +6,47 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 1;
+  let summer = 0;
+
+  const getSum = function(a) {
+    if (counter <= 3 && summer !== 0) {
+      const b = summer;
+
+      counter++;
+      summer = 0;
+
+      return b + a;
+    }
+
+    if (counter % 2 !== 0 && summer !== 0) {
+      const b = summer;
+
+      counter++;
+      summer = 0;
+
+      return b + a;
+    }
+
+    if (counter <= 3 || counter % 2 !== 0) {
+      summer = a;
+
+      return getSum;
+    }
+
+    if (summer === 0) {
+      summer = a;
+
+      return getSum;
+    }
+
+    counter++;
+    summer = 0;
+
+    return 'Bzzz... Error!';
```

Comment:
Lets improve your code. No need in summer variable. I'll give you start suggestion
```suggestion

  const getSum = function(a) {
    return function(b) {
      counter++;
    
      // Make logic here
      
      return a + b;
    }
```